lib/mrb/scripts/command_line/grndb.rb

```diff
@@ -402,6 +405,37 @@ def failed_to_open(name)
           failed(message)
         end
       end
+
+      class Recoverer
+        attr_writer :database
+
+        def initialize
+          @context = Context.instance
+        end
+
+        def recover(force_truncate)
+          @database.recover
+          if force_truncate
+            @database.each do |object|
+              if object.corrupt?
+                logger = @context.logger
+                Dir.foreach(File.dirname(object.path)) do |path|
```

Comment:
In this context, "bad" means that "It is not enough".
object.truncate is not applicable against `VariableSizeColumn` (NoMethodError)
So, we need to implement grn_table_trucate, grn_column_truncate, grn_ii_truncate in mrb, does it correct?